packages/fondue/src/typography/Text/Text.tsx

```diff
@@ -1,8 +1,7 @@
 /* (c) Copyright Frontify Ltd., all rights reserved. */
 
-import { type AriaAttributes, type ReactElement, type ReactNode } from 'react';
-
 import { merge } from '@utilities/merge';
+import { type AriaAttributes, type ReactElement, type ReactNode } from 'react';
```

Comment:
```suggestion
import { type AriaAttributes, type ReactElement, type ReactNode } from 'react';

import { merge } from '@utilities/merge';
```

Reply:
Thnx, did have the latest packages installed